tests/test_coiled.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import uuid
+
+import coiled
+import dask.dataframe as dd
+import pandas as pd
+from dask.distributed import Client
+
+SOFTWARE = f"dask-engineering/coiled_dist-py{sys.version_info[0]}{sys.version_info[1]}"
```

Comment:
@ncclementi just curious, where is this software env built?

should it be versioned (e.g. `0.0.3` for the current version of the runtime)?

Reply:
These were built ad-hoc using environment files that look like

```yaml
channels:
  - conda-forge
  - coiled
dependencies:
    - python=3.9
    - coiled-runtime
```

We've got https://github.com/coiled/coiled-runtime/issues/32 open for formalizing / automating how we manage these software envs